EduVPN/Helpers/CertificateExpiryHelper.swift

```diff
@@ -160,8 +156,10 @@ extension CertificateExpiryHelper.CertificateStatus {
 
     var shouldShowRenewSessionButton: Bool {
         switch self {
-        case .validFor(_, let proportionRemaining):
-            return proportionRemaining < 0.2
+        case .validFor(let timeRemaining):
+            // Show renewal button if session expires in
+            // less than a week.
+            return timeRemaining < (60 * 24 * 7)
```

Comment:
TimeInterval is in seconds, this calculates the number of minutes in a week.

Reply:
Indeed. Good catch, thanks.

Reply:
Fixed in commit 53ca32f.